shared-libs/message-utils/src/index.js

```diff
@@ -14,7 +12,8 @@ var getParent = function(doc, type) {
   while (facility && facility.type !== type) {
     facility = facility.parent;
   }
-  return facility;
+  // don't return falsey values, eg: ""
+  return facility || undefined;
```

Comment:
Just a comment/question. Do we protect from the user entering an empty space or spaces for facility? 
ie. facility: " "
Another question, it seems that we don't define any dependencies for these shared libs; like underscore for this message-utils. Do we just want to avoid collisions?

Reply:
The user doesn't directly set the parent. They select the parent from a dropdown which set it to the object containing the parent id. In this case because the district hospital is the top level contact there is no parent. IMO this should be set to `null` instead of `""`. Configuration will be changed so it will be null from now on but this means it will continue to work with the old style of `parent:""`.

WRT the dependencies of shared libs - I tried to do it for the phone-number shared library and it didn't work correctly - I had to keep reinstalling the node_modules. So now the transitive dependencies are included in the parent package.json and it just works.